src/Symfony/Component/Process/Process.php

```diff
@@ -378,7 +378,7 @@ public function getOutput()
 
         $this->readPipes(false, '\\' === DIRECTORY_SEPARATOR ? !$this->processInformation['running'] : true);
 
-        return $this->stdout;
+        return (string) stream_get_contents($this->stdout, -1, 0);
```

Comment:
casting to string triggers copy-on-writes, it would be better to use an `if` here